README.md

```diff
@@ -120,3 +120,51 @@ const handleLoginClick = async () => {
     await requestOidcAuthentication(app_id, redirect_uri, postLogoutRedirectUri); // If successful, the user will be redirected to the redirectUri
 };
 ```
+
+Once the app has been redirected to the login page and user has entered their credentials, OIDC will redirect the user back to the `redirect_uri` URL that you have specified when calling `requestOidcAuthentication`. The redirect URL will have several new query parameters added automatically, which includes `code` containing the one-time ORY code in the format of `ory_ac...` and the scope which is `openid`.
+Once the user has been redirected to the page, `requestOidcToken` should be called next to retrieve the access tokens. 
+
+```typescript
+// RedirectPage.tsx
+import { requestOidcToken } from '@deriv-com/auth-client'
+
+const RedirectPage = () => {
+    const fetchToken = async () => {
+        const app_id = 'your-app-id'; // The ID of your app
+        const redirect_uri = 'http://your-app/callback'; // The URL to redirect to after successful login
+        const postLogoutRedirectUri = 'http://your-app/'; // The URL to redirect to after logging out
+
+        const { accessToken } = await requestOidcToken(app_id, redirect_uri, postLogoutRedirectUri)
+    }
+
+    useEffect(() => {
+        fetchToken()
+    }, [])
+}
+```
+For the last step, when the access token has been fetched, you will need to call `requestLegacyToken` with the access token passed-in to get the tokens needed to be passed into the `authorize` endpoint.
+
+```typescript
+// RedirectPage.tsx
+import { requestOidcToken, requestLegacyToken } from '@deriv-com/auth-client'
+
+const RedirectPage = () => {
+    const fetchToken = async () => {
+        const app_id = 'your-app-id'; // The ID of your app
+        const redirect_uri = 'http://your-app/callback'; // The URL to redirect to after successful login
+        const postLogoutRedirectUri = 'http://your-app/'; // The URL to redirect to after logging out
+
+        const { accessToken } = await requestOidcToken(app_id, redirect_uri, postLogoutRedirectUri)
+        // Once the access token is returned from `requestOidcToken` and is available, call `requestLegacyToken` to finally retrieve the tokens to pass into authorize
+        const tokens = await requestLegacyToken(accessToken);
+        // You can pass one of the tokens to authorize to login the user
+        callAuthorize(tokens.token1)
```

Comment:
[Q] will we receive multiple tokens for different loginid?

Reply:
Yes, this is in the format of:
```json
{
   token1: ...
   acct1: ...
   token2: ...
   acct2: ...
}
```

Reply:
This is similar to what we have in our current authentication flow, which is the query parameters `?acct1=...,token1=...,acct2=...,token2=...`